client/components/layout/internal/Base/__tests__/Base-spec.js

```diff
@@ -0,0 +1,50 @@
+import React from 'react'
+import expectReactShallow from 'expect-react-shallow'
+
+import Base from '..'
+
+describe('Base', () => {
+  it('renders children within a div by default', () => {
+    expectReactShallow(<Base children={<h1>Test</h1>} type='grid-block' />)
+      .to.have.exactly.rendered(<div className='grid-block'><h1>Test</h1></div>)
```

Comment:
Oh, yeah.  The `div` and `by default` parts threw me off.  I'd probably not mention either of those in this it block, and also not specify the `className` here.  Then I'd add separate specs for `div` by default; other `elementType` when specified.